packages/flutter_tools/lib/src/commands/run.dart

```diff
@@ -308,6 +308,7 @@ class RunCommand extends RunCommandBase {
         initializePlatform: argResults['web-initialize-platform'],
         hostname: featureFlags.isWebEnabled ? argResults['web-hostname'] : '',
         port: featureFlags.isWebEnabled ? argResults['web-port'] : '',
+        skipWebLaunch: featureFlags.isWebEnabled ? argResults['web-skip-launch'] : null,
```

Comment:
The PR title says `--skip-web-launch` but this says `web-skip-launch` - not sure which is intended?

Reply:
Should be --web-skip-launch